health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/components/UserUpload.js

```
@@ -473,34 +474,38 @@ const UserUpload = React.memo(() => {
+                        size="large"
+                        variation="default"
+                        label={t("HCM_VIEW_ERROR")}
+                        onClick={() => setShowPreview(true)}
+                        style={{
+                          marginTop: "1rem",
+                          backgroundColor: "#B91900",
+                        }}
+                        textStyles={{
+                          color: "#FFFFFF",
+                        }}
+                      />,
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)